bin/bootstrap-harmony

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# run harmony from scratch
+
+# check for docker
+if ! command -v "docker" &> /dev/null; then
+  echo "'docker' command not found. Harmony requires a running Docker environment."
+  exit 1;
+fi
+
+# check for kubectl command
+if ! command -v "kubectl" &> /dev/null; then
+  echo "'kubectl' command not found. Harmony requires a running Kubernetes cluster with the 'kubectl' command in your path."
+  exit 1;
+fi
+
+# read EDL credentials from .env file if it exists, otherwise prompt for them
+
+if [[ -f ".env" ]]; then
+  source .env
+else
+  ./bin/create-dotenv
+
+  # remove the empty OUATH env vars already in the file
+  awk -v n=3 'NR==FNR{total=NR;next} FNR==total-n+1{exit} 1' .env .env > env.tmp && mv env.tmp .env
+
+echo << PROMPT
```

Comment:
I'm not seeing this prompt when I run the script.

Reply:
Fixed